prompt.py

```diff
@@ -352,13 +349,13 @@ def do_export(self, arguments):
 
         if item == 'wif':
             if not self.Wallet:
-                return print("please open a wallet")
+                return print("Please open a wallet")
 
             address = get_arg(arguments, 1)
             if not address:
                 return print("Please specify an address")
 
-            passwd = prompt("[Wallet Password]> ", is_password=True)
+            passwd = prompt("[wallet password]> ", is_password=True)
```

Comment:
Since most prompts now start capitalized, `wallet password` here should be `Wallet password`?

Reply:
All prompts start with lower-case characters, only the responses are capitalised sentences.

Would you rather have the prompts capitalised as well? That's somewhat unusual.